drupal/sites/default/boinc/themes/boinc/templates/node-forum.tpl.php

```diff
@@ -100,7 +100,7 @@
     </div>
     <div class="subscribe">
       <?php if ($subscribe_link): ?>
-        <a href="#block-comment_form_block-comment_form">Post new comment</a> |&nbsp;
+        <a href="#block-comment_form_block-comment_form">Post new comment</a>
```

Comment:
I'm guessing there is a corresponding commit to the E@H theme (separate repo) that adds the right border to this "Post new comment" link. I don't yet have that locally, so it appears broken -- this is normal. However, the change does break the BOINC theme, which is in this repo, so that should be addressed.

This can be done either by applying the CSS changes made to the E@H theme also to the BOINC theme, or perhaps more properly using the `<ul>` "links" class and wrapping the subscribe and new comment links in `<li>` elements (each with "first" or "last" class, as appropriate) underneath that `<ul>` container. This should then automatically add the proper border in both themes without any CSS changes being required (see the comment edit / quote links, for example).

Also related to this section of code, it seems questionable that "Post new comment" is rendered only if $subscribe_link is set. In practice, it's probably ok and behaves as expected, but logically the ability to post a comment does not hinge on the subscription link being available. Making them independent does make it a bit trickier to set the right "first" and "last" classes in the `<li>` elements, though, e.g. if there's no subscribe link, the post new comment link is the first and the last link.